scripts/prepdocs.py

```diff
@@ -515,6 +516,27 @@ def read_files(
                 read_files(filename + "/*", use_vectors, vectors_batch_support)
                 continue
             try:
+                # if filename ends in .md5 skip
+                if filename.endswith(".md5"):
+                    print("Skipping md5 hash index.")
```

Comment:
This looks fairly noisy in logs and may confuse folks, do you think its necessary?

Reply:
Not really, removed